### ISSUE TITLE
RMG import errors when running ARC

### ISSUE THREAD

**Issue by alongd** [ReactionMechanismGenerator/ARC#189]
Posted on behalf of @NtuCheRoy:

"When I run my ARC job, it appeared "ImportError: No module named util" error, util is a package of arkane. I installed rmg by "conda install -c rmg rmg", and I had added the PYTHONPATH by "export PYTHONPATH=$PYTHONPATH:~/.conda/envs/arc_env/bin". Could you tell me how to solve this problem?"

**Comment by alongd** [comment]
Currently ARC requires the RMG Developer installation, please see the documentation [here](https://reactionmechanismgenerator.github.io/ARC/installation.html#install-dependencies).

This behaviour might change soon when we transition to Py3.

Please let us know if this did not solve your issue.

**Comment by NtuCheRoy** [comment]
It works, but it appears another error, "ImportError: No module named rmgobject".

**Comment by alongd** [comment]
Did you compile (`activate rmg_env`, `make`)?

**Comment by NtuCheRoy** [comment]
No, I try to run input.yml on your ARC/examples/rates_demo/input.yml, so I use arc_env. Should I compile (activate arc_env, make)?

**Comment by alongd** [comment]
You should compile RMG seperately. ARC does not require compilation. For that, follow the instructions on the RMG documentation: http://reactionmechanismgenerator.github.io/RMG-Py/users/rmg/installation/anacondaDeveloper.html

**Comment by NtuCheRoy** [comment]
When I try to compile, my terminal repeats some lines, as below image, again and again.
![image](https://user-images.githubusercontent.com/53944308/64892533-38108500-d6a7-11e9-9074-105ef1ab235f.png)

**Comment by alongd** [comment]
Compiling RMG can take a while (5-15 minutes?), depending on the machine you're using. Meanwhile you'll see many compilation massages. You should wait to the end of this process when it says something like `everything found :)`

**Comment by NtuCheRoy** [comment]
Ok ! I finished compilation of RMG. When I run input.yml mentioned in previous comments, it appear another error “ ImportError: No module nameed autotst.reation. I had added ARC file to PYTHONPATH. Do you know how to fix it out? Thank you!

**Comment by NtuCheRoy** [comment]
I finished the compilation of RMG. However, when I run input.yml mentioned in previous comment, it appeared another error "ImportError: cannot import name autotst.reaction". I installed autotst software and add it in my PYTHONPATH, but appeared another error "ImportError: cannot import name AutoTST_Reaction". Then, I check the AutoTST/autotst/reaction.py file, it seems that there doesn't have something relative to AutoTST_Reaction. Do you know how to solve this problem?

**Comment by alongd** [comment]
Can you share the error trace message you got?

**Comment by NtuCheRoy** [comment]
![image](https://user-images.githubusercontent.com/53944308/64912531-50de7080-d763-11e9-9622-5c64cbe73aa2.png)
By the way, if I just want to run my job on server by using qchem built on server as Electronic Structure Software. Is that right I just type below code in my yml file?
`ess_settings:
  qchem:
  - server1`
or it should be 'local' ?

**Comment by alongd** [comment]
The AutoTST error is indeed a bug, will be soon fixed via #191. Thanks for reporting!
Seems like you have another unrelated error in your run (`KeyError: 'TSO'`). Is `TSO` a species in your input file? If this is still a problem, let's discuss it in a separate GitHub issue.
I'll address your input file question in a separate issue as well (so future users could more conveniently find answers)

Once #191 is merged, update ARC. Let me know if AutoTST still causes issues.
